src/coreComponents/mesh/utilities/MeshMapUtilities.hpp

```diff
@@ -57,21 +49,25 @@ inline localIndex size0( arrayView2d< T, USD > const & map )
 }
 
 /**
- * @copydoc size0(arrayView1d< arrayView1d< T const > const > const &)
+ * @brief @return the size of the map along first dimension
+ * @tparam T type of map element
+ * @param map reference to the map
  */
 template< typename T >
 GEOSX_HOST_DEVICE
-inline localIndex size0( ArrayOfArraysView< T const > const & map )
+inline localIndex size0( ArrayOfArraysView< T > const & map )
```

Comment:
It feels like we shouldn't have these functions?

Reply:
Yes, but they are currently necessary because `ArrayOfArrays::size()` and `array2d::size()` mean different things. I explained in a bit more detail and made a proposal here: https://github.com/GEOSX/LvArray/issues/253 but that change should be its own PR.

Reply:
I had forgot about this little piece of ugliness.